backend/app/services/knowledge/knowledge_service.py

```
@@ -1513,6 +1514,86 @@ def get_accessible_knowledge(
+        Returns:
+            Dict with 'created_by_me' and 'shared_with_me' lists
+        """
+        from app.models.resource_member import MemberStatus, ResourceMember
+        from app.models.share_link import ResourceType
+
+        # Get KBs created by user (personal knowledge bases, namespace=default)
+        created_kbs = (
+            db.query(Kind)
+            .filter(
+                Kind.kind == "KnowledgeBase",
+                Kind.is_active == True,
```

Ruff (0.15.5)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)